Documentation/helm-operator.md

```diff
@@ -58,16 +69,21 @@ To install the helm chart from master, you will need to pass the specific versio
 
 ```console
 helm repo add rook-master https://charts.rook.io/master
+# For Helm v3.x
+helm search repo rook-ceph --versions
+# For Helm v2.x
 helm search rook-ceph
-helm install --namespace rook-ceph rook-master/rook-ceph --version <version>
+helm install --namespace rook-ceph rook-ceph rook-master/rook-ceph --version <version>
```

Comment:
How about adding a comment before this line? Without a comment it looks like this line only applies to Helm 2.x

Reply:
I added some spacing as well to make it clearer.